Testscripts/Linux/SetupRDMA.sh

```diff
@@ -68,7 +68,11 @@ function Main() {
 		redhat_7|centos_7|redhat_8|centos_8)
 			# install required packages regardless VM types.
 			LogMsg "Starting RDMA setup for RHEL/CentOS"
-			req_pkg="kernel-devel-$(uname -r) valgrind-devel redhat-rpm-config rpm-build gcc gcc-gfortran libdb-devel gcc-c++ glibc-devel zlib-devel numactl-devel libmnl-devel binutils-devel iptables-devel libstdc++-devel libselinux-devel elfutils-devel libtool libnl3-devel java libstdc++.i686 gtk2 atk cairo tcl tk createrepo byacc.x86_64 net-tools"
+			# Due to redhat bug, 1787637, this workaround is required.
+			suppliment_pkg="dnf rpm"
```

Comment:
supplement

Reply:
fixed in 87b3b5cb1ba0d82afd52adf2a331f547e1fb0443